charts/cryostat/templates/deployment.yaml

```diff
@@ -190,6 +190,23 @@ spec:
           securityContext:
             {{- toYaml .Values.storage.securityContext | nindent 12 }}
         {{- if not .Values.minimal }}
+        - name: {{ printf "%s-%s" .Chart.Name "oauth2proxy" }}
+          securityContext:
+            {{- toYaml .Values.oauth2Proxy.securityContext | nindent 12 }}
+          image: "{{ .Values.oauth2Proxy.image.repository }}:{{ .Values.oauth2Proxy.image.tag }}"
+          imagePullPolicy: {{ .Values.oauth2Proxy.image.pullPolicy }}
+          env:
+            - name: CRYOSTAT_OAUTH2PROXY_URL
```

Comment:
I don't think the name of this environment variable should have been changed. The oauth2-proxy container won't know how to interpret a variable named like this.